BedrockServer.cpp

```diff
@@ -1520,8 +1516,9 @@ bool BedrockServer::_isStatusCommand(BedrockCommand& command) {
 list<STable> BedrockServer::getPeerInfo() {
     SAUTOLOCK(_syncMutex);
     list<STable> peerData;
-    if (_syncNode) {
-        for (SQLiteNode::Peer* peer : _syncNode->peerList) {
+    auto _syncNodeCopy = _syncNode;
```

Comment:
just curious -- why do we need this copy now?

Reply:
Actually, I lied, there is a `_standDownTimeOut` which will force it to stand down non-gracefully after 30 seconds, the same as for shutting down. Though if we hit that, we will drop commands and have unpredictable behavior.

Reply:
Gotcha. This is pretty much what I thought, thanks for clarifying!

Reply:
Weird, I added a follow up comment but it's not appearing here (watch this comment appear below a nearly identical comment when I submit it), but I lied, there is a `_standDownTimeOut` alarm that will force a non-graceful stand down after 30 seconds, just like for shutting down. If we hit that, it means we will drop commands and lose stuff.